### DISCUSSION TITLE
PoPageDynamicTableFilters - Disclaimer Group não dispara evento validate()

### DISCUSSION THREAD

**Discussion by h2danilo** [po-ui/po-angular#1924] [Tire suas dúvidas]
Boa tarde,

Estou enfrentando um problema com o componente po-page-dynamic-table no PO UI. Ao utilizar este componente, percebi que o evento de validação (validate)  não é acionado para uma determinada propriedade (detalhaSaldo) quando o filtro de pesquisa é limpo através do grupo de Disclaimers (Disclaimer Group).

Estou buscando uma solução para este comportamento. Especificamente, implementei uma validação que altera a exibição do p-table-custom-actions com base no valor de uma propriedade. Quando modifico o valor no filtro de pesquisa, a validação funciona como esperado e o p-table-custom-actions é exibido ou ocultado corretamente. No entanto, ao limpar o filtro utilizando o grupo de Disclaimers, o evento de validação não é disparado, o que me impede de saber se devo ou não exibir o p-table-custom-actions.

Poderiam me fornecer orientações sobre como resolver ou contornar este problema?

Agradeço antecipadamente pela atenção e suporte.

Envio código-fonte abaixo:
[po-page-dynamic-table.txt](https://github.com/po-ui/po-angular/files/13942730/po-page-dynamic-table.txt)
[component.txt](https://github.com/po-ui/po-angular/files/13942733/component.txt)

**Comment by anliben** [comment]
Boa Tarde @h2danilo, tudo bem?

Iremos analisar e em breve iremos responder sua solicitação.

attr.
